src/loki/reach.cc

```diff
@@ -29,10 +29,15 @@ void Reach::enqueue(const baldr::GraphId& node_id,
   // otherwise we enqueue it
   queue_.insert(node_id);
   // and we enqueue it on the other levels
-  for (const auto& transition : tile->GetNodeTransitions(node))
+  for (const auto& transition : tile->GetNodeTransitions(node)) {
+    // skip nodes which are done already
+    if (done_.find(transition.endnode()) != done_.cend())
```

Comment:
if the transition node was already seen and expanded we dont want to do it again or else that will over count